torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -41,6 +41,9 @@ int64_t SizeNode::getDynamicValue() const {
   }
   torch::lazy::NodePtr cloned =
       torch::lazy::MakeNode<SizeNode>(operands_[0], dim_);
+  // Wrap the IR of SizeNode into a dummy tensor and execute/fetch the value
+  // of this tensor. GetTensors will return a cpu at::Tensor so we can just
```

Comment:
If it is a TPU at::Tensor, we cannot extract the value of it?

Reply:
if it is a TPU tensor, it must be a `XLATensor`